WARNING: fsync() not supported in selected backend.  Using value of 0.
IOR-3.4.0+dev: MPI Coordinated Test of Parallel I/O
Began               : Sun Dec 13 12:02:18 2020
Command line        : ./ior -Q 1 -F -g -G 4711 -z --random-offset-seed=11 -e -o ./datafiles/2020.12.13-11.48.46/ior-rnd/file -O stoneWallingStatusFile=./results/2020.12.13-11.48.46/ior-rnd.stonewall -O stoneWallingWearOut=1 -t 4096 -b 1073741824 -s 10000000 -w -D 300 -a S3-libs3 --S3-libs3.host=storage.googleapis.com --S3-libs3.secret-key=foo --S3-libs3.access-key=foo --S3-libs3.s3-compatible --S3-libs3.use-ssl --S3-libs3.location=EU -k
Machine             : Linux mg100
WARNING: S3 S3_init:512 (path:ior777) "ErrorInternalError": We encountered an internal error. Please try again. 
TestID              : 0
StartTime           : Sun Dec 13 12:02:19 2020
Path                : ./datafiles/2020.12.13-11.48.46/ior-rnd/file.00000000
FS                  : 0.0 GiB   Used FS: 0.0%   Inodes: 0.0 Mi   Used Inodes: -inf%

Options: 
api                 : S3-libs3
apiVersion          : 0.5
test filename       : ./datafiles/2020.12.13-11.48.46/ior-rnd/file
access              : file-per-process
type                : independent
segments            : 10000000
ordering in a file  : random
ordering inter file : no tasks offsets
nodes               : 2
tasks               : 2
clients per node    : 1
repetitions         : 1
xfersize            : 4096 bytes
blocksize           : 1 GiB
aggregate filesize  : 19531.25 TiB
stonewallingTime    : 300
stoneWallingWearOut : 1

Results: 

access    bw(MiB/s)  IOPS       Latency(s)  block(KiB) xfer(KiB)  open(s)    wr/rd(s)   close(s)   total(s)   iter
------    ---------  ----       ----------  ---------- ---------  --------   --------   --------   --------   ----
stonewalling pairs accessed min: 2780 max: 2797 -- min data: 0.0 GiB mean data: 0.0 GiB time: 300.1s
WARNING: Expected aggregate file size       = 21474836480000000
WARNING: Stat() of aggregate file size      = 0
WARNING: Using actual aggregate bytes moved = 22913024
WARNING: Maybe caused by deadlineForStonewalling
write     0.072361   18.53      0.001151    1048576    4.00       0.116498   301.86     0.007399   301.98     0   
WARNING: S3 S3_final:524 (path:ior777) "ErrorBucketNotEmpty": The bucket you tried to delete is not empty. 
